frontend/e-learning-fe/src/components/notAdminUsers.js

```diff
@@ -0,0 +1,92 @@
+import React, { useEffect, useState } from "react";
+
+import API from "../api";
+
+function NotAdminUsers() {
+  const [users, setUsers] = useState([]);
+  const [loading, setLoading] = useState(false);
```

Comment:
change to `isLoading` the variable is boolean so lets add `is`, that way we instinctively know that this is only `true` or `false`